src/steamship/agents/memory/chathistory.py

```diff
@@ -160,3 +163,30 @@ def messages(self) -> List[Block]:
     @property
     def client(self) -> Client:
         return self.file.client
+
+    def messages_as_string(
+        self,
+        memory_window_strategy: MemoryStrategy,
+        user_prefix: str = "User",
+        assistant_prefix: str = "AI",
+    ) -> str:
+        messages = memory_window_strategy.messages(self)
+        print(f"messages: {messages}")
+        as_strings = []
+        for block in messages:
+            role = block.chat_role
+            if role == RoleTag.USER:
+                as_strings.append(f"{user_prefix}: {block.text}")
```

Comment:
When would `user_prefix` and `assistant_prefix` need to change? I would make this static for now.